src/tests/ftest/osa/offline_reintegration.py

```diff
@@ -8,7 +8,7 @@
 from osa_utils import OSAUtils
 from daos_utils import DaosCommand
 from nvme_utils import ServerFillUp
-from test_utils_pool import TestPool
+from test_utils_pool import add_pool
```

Comment:
(pylint-syntax-error) Cannot import 'test_utils_pool' due to syntax error 'invalid syntax (<unknown>, line 53)'